frontend/catalyst/from_plxpr.py

```diff
@@ -493,6 +561,38 @@ def eval(self, jaxpr: "jax.core.Jaxpr", consts: Sequence, *args) -> list:
         return outvals
 
 
+class PredicatePlxprInterpreter(PlxprInterpreter):
+    """An interpreter that converts a plxpr predicate into catalyst-variant jaxpr branch."""
+
+    # pylint: disable=too-many-branches
```

Comment:
😕 ?
How is this function having too many branches? I'm not really seeing any branches?